AdminWebsite/AdminWebsite/Controllers/WorkHoursController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using BookingsApi.Client;
+using BookingsApi.Contract.Requests;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace AdminWebsite.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/workhours")]
+    public class WorkHoursController : ControllerBase
+    {
+
+        private readonly IBookingsApiClient _bookingsApiClient;
+
+        public WorkHoursController(IBookingsApiClient bookingsApiClient)
+        {
+            _bookingsApiClient = bookingsApiClient;
+        }
+
+        [HttpPost]
+        [AllowAnonymous]
+        [SwaggerOperation(OperationId = "UploadWorkHours")]
+        [ProducesResponseType(typeof(List<string>), (int)HttpStatusCode.OK)]
```

Comment:
rather than returning a list of strings, could we maybe return an `UploadWorkHoursResponse` object instead, with a `List<string> FailedUsernames` as a property?

it would make it easier to change later if we need to return additional information